public/diagnostic.js

```diff
@@ -7,7 +7,33 @@
 //  -  a unit price
 //  -  a quantity
 //  -  a computed property called `orderPrice`, equal to price times quantity
-//
+
+const Order = Ember.Object.extend({
+    unitPrice: '',
+    quantity: '',
+    orderPrice: Ember.computed('unitPrice', 'quantity', function(){
+      return this.get('unitPrice') * this.get('quantity');
+    })
+});
+
+const Cart = Ember.Object.extend({
+  orders: [],
+  addToCart: function(){
+    return
+  }
+  totalPrice: Ember.computed('orderPrice')
```

Comment:
This is not the correct setup. A custom computed property needs a function to determine its value. Additionally, this property is watching `orderPrice` with is not a property that a Cart has - it's a property of the Orders.